incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -138,23 +138,28 @@ namespaces.
 
 <a name="basic-trees">
 
-### Parents, children, trees and forests
-
-When using HNC, every namespace may have either zero or one parent. A namespace
-with no parents is known as a _root_ namespace, while all namespaces with the
-same parent are known as the _children_ of that namespace. Non-root namespaces
-may have children of their own; there is no hard limit to the number of levels
-of hierarchy.
-
-All namespaces that descend from a root namespace, along with the root itself,
-are called a _tree_. The set of all namespaces that descend from any namespace,
-along with that namespace, is called a _subtree_. Namespaces without children
-are called _leaves_. Note that a leaf namespace is technically also a subtree,
+### Parents, children, trees, and forests
```

Comment:
Probably add 'root' to the title? 'ancestors' and 'descendants' may be ok since already having parent, children.

Reply:
I didn't want to make this longer so I shortened it to "Namespaces, trees, and forests."

Reply:
sg this is even better!